.github/workflows/create_new_release.yml

```diff
@@ -0,0 +1,34 @@
+name: Create new release
+
+on:
+  pull_request:
+    types: [ closed ]
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    if: github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'release')
```

Comment:
This job can be triggered by any merged PR with a specific label (which can be added manually). (I can create PR compromised PR, add release label and merge it. If I understand the job description correctly -- it will start release)

If I am not wrong, release notes PR is created by a bot, I think we should consider adding an additional condition which verifies who is the creator or merged PR.

It's the first that came to my mind, if there is a better way to make it more secure I think it would be worth to implement it.

Of course, let me know, I might be wrong and this implementation is already secure 💪

Reply:
Yeah, you are 100% right that if someone uses `release` label it will generate new release,
I think that we could make additional check if issue is created by bot and/or check if title starts with `chore: release notes for`